pkg/base-dev/lib/docket.hoon

```diff
@@ -79,10 +84,12 @@
         ==
         ?~  image.d  ~  ~[image+u.image.d]
         ?:  ?=(%site -.href.d)  ~[site+path.href.d]
+        =/  loc=glob-location  glob-location.href.d
         :~  base+base.href.d
-            glob+url.glob-location.href.d
-        ==
-    ==
+            ?-  -.loc
+              %http  [%glob-http url.loc]
+              %ship  [%glob-ames ship.loc]
```

Comment:
now that I'm looking at this, I think we'd be better off using %ames instead of %ship, since then both tags are names of protocols, and there's a nice symmetry with %http->%glob-http and %ames->%glob-ames